skll/config/__init__.py

```diff
@@ -254,122 +257,168 @@ def parse_config_file(config_path, log_level=logging.INFO):  # noqa: C901
     ----------
     config_path : str
         The path to the configuration file.
-    log_level : logging level, optional
+
+    log_level : logging level, optional, default=logging.INFO
```

Comment:
```suggestion
    log_level : logging level, default=logging.INFO
```

We should change to an actual type, not just "logging level" and since we are specifying `default` now, we don't need the word "optional" any more. I think that should be implied.

Reply:
Makes sense! Technically, `log_level` is an integer. However, it's more common to pass `log_level` one of the `logging` parameters, like `logging.INFO` or `logging.WARNING`. I think that might be worth including in the description. Here's an example:

```
    log_level : int, default=logging.INFO
        Determines which messages should be logged. You can either pass
        an ``int`` or one of the corresponding ``logging`` parameters, such
        as ``logging.INFO`` or ``logging.WARNING``. 
```

What are your thoughts?

Reply:
Great idea! Let's do it!